nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -447,68 +452,61 @@ Costmap2DROS::getOrientedFootprint(std::vector<geometry_msgs::msg::Point> & orie
 }
 
 void
-Costmap2DROS::mapUpdateLoop(double frequency)
+Costmap2DROS::mapUpdateLoop()
 {
-  RCLCPP_DEBUG(get_logger(), "mapUpdateLoop frequency: %lf", frequency);
+  static const double expected_period = 1. / map_update_frequency_;
```

Comment:
`map_update_frequency_` should be checked to be `!= 0.0` to avoid division by zero

Reply:
ok